src/components/shared/Line/Line.js

```diff
@@ -21,7 +21,7 @@ export default function Line({
   yAxisId,
 }) {
   const label = showDataAnnotations ? (
-    <DataAnnotation valueFormatter={valueFormatter} />
+    <DataAnnotation valueFormatter={valueFormatter} position="top" shouldUseSpaceOffset={false} />
```

Comment:
The math for picking the position of the label is different between `line` and `bar`, because for `bar`, we have to take into account the width of the bars to center the label within the bar, but for line, we don't need to care about the width. I'm trying to encapsulate that difference in `shouldUseSpaceOffset`, which is `false` for line and `true` for bar plots.